pkgs/applications/science/electronics/horizon-eda/default.nix

```
@@ -0,0 +1,69 @@
+{ stdenv
+, boost
+, coreutils
+, cppzmq
+, curl
+, epoxy
+, fetchFromGitHub
+, glm
+, gnome3
+, lib
+, libgit2
+, librsvg
+, libuuid
+, libzip
+, opencascade
+, pkgconfig
+, podofo
+, python3
+, sqlite
+, wrapGAppsHook
+, zeromq }:
+
+stdenv.mkDerivation rec {
+  pname = "horizon-eda";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "horizon-eda";
+    repo = "horizon";
+    rev = "v${version}";
+    sha256 = "0b1bi99xdhbkb2vdb9y6kyqm0h8y0q168jf2xi8kd0z7kww8li2p";
+  };
+
+  buildInputs = [
+    boost
+    cppzmq
+    curl
+    epoxy
+    glm
+    gnome3.gtkmm
+    libgit2
+    librsvg
+    libuuid
+    libzip
+    opencascade
+    podofo
+    python3
+    sqlite
+    zeromq
+  ];
+
+  nativeBuildInputs = [ pkgconfig wrapGAppsHook ];
+
+  NIX_CFLAGS_COMPILE = "-I${opencascade}/include/oce";
```

Looking at the `Makefile` it seems like:
```suggestion
  CASROOT = opencascade;
```

is better. This is also suggested by upstream: https://old.opencascade.com/doc/occt-7.0.0/overview/html/

---

Also see #79303